tensorflow_federated/python/research/robust_aggregation/BUILD

```diff
@@ -0,0 +1,35 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])  # Apache 2.0
+
+py_library(
+    name = "robust_aggregation",
+    srcs = ["__init__.py"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":rfa",
+    ],
+)
+
+py_library(
+    name = "rfa",
```

Comment:
Could we rename this "robust_aggregation.py" so that the implementation/test match the "file.py/file_tes.tpother places in TFF?

Reply:
Done!